virtcontainers/api.go

```diff
@@ -39,29 +39,53 @@ func CreateSandbox(sandboxConfig SandboxConfig) (VCSandbox, error) {
 }
 
 func createSandboxFromConfig(sandboxConfig SandboxConfig) (*Sandbox, error) {
+
```

Comment:
Could you remove the blank line before this variable (same comment for the function below)?

Reply:
Thanks for your careful review, and I have removed the blank line